diff -ur lsp-plugins-1.1.28/src/doc/inc/plugins.php lsp-plugins-1.1.28n/src/doc/inc/plugins.php
--- lsp-plugins-1.1.28/src/doc/inc/plugins.php	2020-12-20 22:10:45.000000000 +0000
+++ lsp-plugins-1.1.28n/src/doc/inc/plugins.php	2020-12-21 14:37:16.732659027 +0000
@@ -5,7 +5,8 @@
 	{
 		$a = $a['description'];
 		$b = $b['description'];
-		return ($a == $b) ? 0 : ($a > $b) ? 1 : -1;
+		 return ($a == $b) ? 0 : (($a > $b) ? 1 : -1);
+
 	}
 	
 	function sort_plugins()
